lms/djangoapps/courseware/tests/test_self_paced_overrides.py

```diff
@@ -118,31 +118,31 @@ def test_course_access_to_beta_users(self):
         self.assertTrue(has_access(beta_tester, 'load', self_paced_section, self_paced_course.id))
 
     @patch.dict('courseware.access.settings.FEATURES', {'DISABLE_START_DATES': False})
-    def test_instructor_paced_discussion_module_visibility(self):
+    def test_instructor_paced_discussion_xblock_visibility(self):
         """
-        Verify that discussion modules scheduled for release in the future are
+        Verify that discussion xblock scheduled for release in the future are
         not visible to students in an instructor-paced course.
         """
         course, section = self.setup_course(start=self.now, self_paced=False)
-        self.create_discussion_modules(section)
+        self.create_discussion_xblocks(section)
 
         # Only the released module should be visible when the course is instructor-paced.
```

Comment:
Nit: You did such a thorough job with renaming, that you could renamed 'module(s)' to 'xblock(s)' on from here to the end of this method.